src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.td

```diff
@@ -88,6 +88,17 @@ def replaceONNXBatchNormalizationInferenceModePattern : Pattern<
 // 
 //===----------------------------------------------------------------------===//
 
+// Get a type for a tensor that stores the shape of another tensor.
+def GetStart: NativeCodeCall<
+  //"IntegerType::get($_builder, 64, IntegerType::Signed)"
```

Comment:
Should remove commented code. Have you though about adding a custom builder of the operation that does not have the start/end and would automatically generate them.... instead of modifying each of the patterns involved with the new shape op?

Reply:
I did consider adding a custom builder. In all honesty I tried and was having some issues getting it to work.
 
 But you makes good points.  I have added a Native code call to more succinctly create ONNX Shape ops
 ```
 // Create an ONNX Shape Op with type
def CreateShapeOp: NativeCodeCall<
  "$_builder.create<mlir::ONNXShapeOp>($_loc, $0, $1, IntegerAttr(), 0)"
>;
```

And I now call this, to avoid having to use the start and end attributes everywhere.

Reply:
Awesome, glad you found a way to simplify the code